app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
Hm, but it appears in the HTML then
![screen shot 2017-06-14 at 5 03 26 pm](https://user-images.githubusercontent.com/173537/27159658-80901ba8-5123-11e7-90c6-73069925dec1.png)